### DISCUSSION TITLE
⚠️ No farming here, only meta discussions!

### DISCUSSION THREAD

**Discussion by Schweinepriester** [Schweinepriester/github-profile-achievements#722] [General]
**EDIT 2024-02-15**: GitHub [announced](https://github.com/orgs/community/discussions/106536) that Achievements will no longer be available in [GitHub’s Community Discussions](https://github.com/orgs/community/discussions/), so don't spam there and do actually useful things organically.

**TL;DR**: Go to https://github.com/Schweinepriester/achievement-testarea/discussions if you have to exchange stars, answers, etc.

As GitHub currently doesn't allow fine grained notifications I can't take the discussion notifications spam anymore.

Please actually contribute, experiment in your own repos or if you must, go to https://github.com/Schweinepriester/achievement-testarea/discussions to exchange stars, answers etc.

If this pin is ignored, I'm going to close down discussions.

For now, I'm leaving discussions open for meta discussions about the achievements, e.g. if there are new ones.

**Comment by Veercodeprog** [comment]
i appreciate your stand.why not close other discussions as well.

**Reply by Schweinepriester** [reply]
I will, eventually!
Started recently with the soft nudging…

**Comment by Veercodeprog** [comment]
promoting repos is other thing,
singing star for a star looks pretty unethical.
good initiative.

**Comment by ArielMAJ** [comment]
Okay! I'll keep this in mind! I'll be migrating to that other repo for star exchanging and stop doing it here.

**Comment by DhrubajyotiBiswasfounderofDribll** [comment]
Okay!i will keep this in my mind.

**Comment by lqq-code** [comment]
all right

**Comment by Schweinepriester** [comment]
Just adding a comment so it's on the top again 🙃

**Comment by bilalmirza74** [comment]
Test Pass

**Comment by Noctino52** [comment]
Test pass

**Comment by the1Riddle** [comment]
:+1: understood ^-^ ^_^

**Comment by rafidRD** [comment]
yess done

**Comment by hypebeans** [comment]
Understood! Will do! 🫶
